composer.json

```diff
@@ -7,7 +7,7 @@
     "repositories": [
         {
             "type": "vcs",
-            "url": "https://github.com/laravel-shift/laravel-timezone.git"
+            "url": "https://github.com/ZTL-ARTCC/laravel-timezone.git"
```

Comment:
I don't really like this practice of pulling away from the official package, but in this case I see this hasn't been updated in 2 years so I'm ok with it. However, if this gets updated to support v12, I think we should switch back to that one.